Docker/Dockerfile

```
@@ -0,0 +1,76 @@
+FROM ubuntu:24.04
+
+# Install prerequisites (including gcc, g++, glibc, cmake, make, ninja, and python3)
+# Latest available gcc and g++ are installed - should we be specific about the version?
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+    lsb-release \
+    software-properties-common \
+    gnupg \
+    curl \
+    git \
+    build-essential \
+    cmake \
+    ninja-build \
+    python3 \
+    python3-pip \
+    python3-venv \
+    wget
+
+# Install LLVM 20
+RUN wget https://apt.llvm.org/llvm.sh && \
+    chmod +x llvm.sh && \
+    ./llvm.sh 20 && \
+    rm -rf llvm.sh
+
+RUN ln -s /usr/bin/clang-20 /usr/bin/clang && \
+    ln -s /usr/bin/clang++-20 /usr/bin/clang++
+
+ENV CC=clang
+ENV CXX=clang++
+
+# Install .NET Core SDK and Runtime - 9.0
+RUN add-apt-repository ppa:dotnet/backports
+
+RUN apt-get install -y \
+    dotnet-sdk-9.0 \
+    dotnet-runtime-9.0
```

.NET SDK includes .NET runtime
```suggestion
    dotnet-sdk-9.0
```

---

Fixed. Thank you.